### PR TITLE
File Added : JS Assignments 5, 6, 9

### PR DESCRIPTION
**Pull request by Oshin-250197**

### CODE REVIEW

**Review by roshnidurugwar — APPROVED** [review]

File: Oshin1/JavaScript/Assignment5/Fibonacci_series.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>FABONACI SERIES</title>
+        <style>
+.container {
```
Comment (Pawan1005) [inline Oshin1/JavaScript/Assignment5/Fibonacci_series.html]: make correct indentation

File: Oshin1/JavaScript/Assignment5/Fibonacci_series.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            }
+            #pid {
+                color: white;
+                text-align: center;
+                font-size: large;
+            }
+        </style>
+    </head>
+    <body >
+
```
Comment (Pawan1005) [inline Oshin1/JavaScript/Assignment5/Fibonacci_series.html]: remove extra space from body tag 
and remove extra line

File: Oshin1/JavaScript/Assignment5/Fibonacci_series.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                                        n2 = n3;
+                                        num--;
+                                    }
+                                }
+                            </script>
+
+                </div>
+            </div>
+
+</html>
```
Comment (Pawan1005) [inline Oshin1/JavaScript/Assignment5/Fibonacci_series.html:75]: body tag closing missing 
make proper indentation

File: Oshin1/JavaScript/Assignment9/Prime_Palin.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+                                document.getElementById("num2").innerHTML += " &emsp; is &nbsp;<br><br>";
+                                document.getElementById("pid").innerHTML += "  Prime Palindrome <br><br>";
+                            } else {
+                                document.getElementById("num1").innerHTML += +str + "<br><br>";
+                                document.getElementById("num2").innerHTML += "&emsp; is &nbsp;<br><br>";
+                                document.getElementById("pid").innerHTML += " Not a Prime Palindrome <br><br>";
+                            }
+                        }
+                        var string = [],
+                            i;
```
Comment (Pawan1005) [inline Oshin1/JavaScript/Assignment9/Prime_Palin.html]: put this on same line

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by arshad05786 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**